back/engines/commercial/admin_api/app/graphql/admin_api/types/initiative_type.rb

```diff
@@ -41,6 +41,7 @@ def large_url
     field :href, String, null: true
     field :upvotes_count, Integer, null: false
     field :comments_count, Integer, null: false
+    field :internal_comments_count, Integer, null: false
```

Comment:
Not entirely sure this is necessary, but working on assumption it doesn't hurt (for now)